id-repository/credential-service/src/test/java/io/mosip/credentialstore/test/provider/impl/CredentialProviderTest.java

```diff
@@ -115,6 +138,1007 @@ public void setUp() throws DataEncryptionFailureException, ApiNotAccessibleExcep
+			kyc1.setAttributeName("fullName");
+			kyc1.setEncrypted(true);
+
+			List<Source> sourceList = new ArrayList<>();
+
+			Source source1 = new Source();
+			source1.setAttribute("fullName");
+
+			sourceList.add(source1);
+			kyc1.setSource(sourceList);
+			shareableAttributes.add(kyc1);
+			AllowedKycDto kyc2 = new AllowedKycDto();
+			kyc2.setAttributeName("dateOfBirth");
+			kyc2.setEncrypted(true);
+
+			List<Source> sourceList2 = new ArrayList<>();
+			Source source2 = new Source();
+			source2.setAttribute("dateOfBirth");
+			sourceList2.add(source2);
+			kyc2.setSource(sourceList2);
+			shareableAttributes.add(kyc2);
+			AllowedKycDto kyc3 = new AllowedKycDto();
+			kyc3.setAttributeName("biometrics");
+			kyc3.setGroup("CBEFF");
+			kyc3.setEncrypted(true);
+			List<Source> sourceList3 = new ArrayList<>();
+			Source source3 = new Source();
+			source3.setAttribute("individualBiometrics");
+			sourceList3.add(source3);
+			kyc3.setSource(sourceList3);
+			shareableAttributes.add(kyc3);
+			AllowedKycDto kyc4 = new AllowedKycDto();
+			List<Source> sourceList4 = new ArrayList<>();
+			Source source4 = new Source();
+			source4.setAttribute("email");
+			sourceList4.add(source4);
+			kyc4.setSource(sourceList4);
+			kyc4.setAttributeName("email");
+			kyc4.setEncrypted(true);
+			kyc4.setGroup(CredentialConstants.CBEFF);
+			shareableAttributes.add(kyc4);
+			PolicyAttributesDto dto = new PolicyAttributesDto();
+			dto.setShareableAttributes(shareableAttributes);
+			policyResponse.setPolicies(dto);
+		}
+	}
+
+	@Test
+	public void testGetFormattedCredentialDataSuccess() throws CredentialFormatterException {
+		CredentialServiceRequestDto credentialServiceRequestDto = new CredentialServiceRequestDto();
+		Map<String, Object> additionalData = new HashMap<>();
+		credentialServiceRequestDto.setAdditionalData(additionalData);
+		Map<AllowedKycDto, Object> sharableAttributes = new HashMap<>();
+
+		AllowedKycDto kyc1 = new AllowedKycDto();
+		kyc1.setAttributeName("fullName");
+		kyc1.setEncrypted(true);
+		List<Source> sourceList = new ArrayList<>();
+		Source source1 = new Source();
+		source1.setAttribute("fullName");
+
+		sourceList.add(source1);
+		kyc1.setSource(sourceList);
+
+		AllowedKycDto kyc2 = new AllowedKycDto();
+		kyc2.setAttributeName("biomterics");
+		kyc2.setEncrypted(true);
+		kyc2.setGroup("CBEFF");
+		List<Source> sourceList1 = new ArrayList<>();
+		Source source2 = new Source();
+		source2.setAttribute("individualBiometrics");
+
+		sourceList1.add(source2);
+		kyc2.setSource(sourceList1);
+		sharableAttributes.put(kyc1, "testname");
+		sharableAttributes.put(kyc2, "biomtericencodedcbeffstring");
+		DataProviderResponse dataProviderResponse = credentialDefaultProvider
+				.getFormattedCredentialData(
+						credentialServiceRequestDto, sharableAttributes);
+		assertNotNull(dataProviderResponse);
+	}
+
+	@Test
+	public void testGetFormattedCredentialData() throws CredentialFormatterException {

```

Comment:
please follow the naming convention for test case name 
eg: createBinder_withApplicationConversionService_returnSuccessResponse